docs/update-from-client-side-helpers-to-server-side.md

```diff
@@ -0,0 +1,229 @@
+
+```js
+createOrder(data, actions) {
+  return actions.order.create({
+    purchase_units: [
+      {
+          amount: {
+            value: "88.44",
+          },
+      }],
+  });
+}
+```
+### Changes needed to migrate to using server-side integration pattern
+
+To simplify the integration of your e-commerce website with the PayPal v2 Orders API, you can move the order creation process to your server-side. The following steps are required to create an order on the server-side:
+
+1. Obtain an access token to use for PayPal backend API calls. This [video tutorial](https://www.youtube.com/watch?v=HOkkbGSxmp4&t=113s) can walk you through the steps. 
+2. Pass necessary checkout info from the browser client to your server-side API endpoint.
+3. Call the PayPal Orders API from your server-side code and return the order ID in your createOrder() callback. 
+
+## New state of client + server integration behavior
+
+```js
+createOrder: function (data, actions) {
+  return (
+      // 1. send your cart info to your server side to create a PayPal Order.
+      fetch("/your-server/api/create-paypal-order", {
+          method: "POST",
+          body: JSON.stringify({
+          //  EXAMPLE DATA ONLY 
+            cart: [
+              {
+                sku: "YOUR_PRODUCT_STOCK_KEEPING_UNIT",
+                quantity: "YOUR_PRODUCT_QUANTITY",
+              }],
+            }),
+        })
+        .then((response) => response.json())
+        // 2. return the PayPal Order ID that you received from the PayPal backend
+        .then((order) => order.id)
+      );
+}
+```
+
+**The following diagram shows the process of creating an order:**
+
+```mermaid
+sequenceDiagram
+    actor Buyer
+    participant M(HTML) as Merchant HTML Page
+    participant M(S) as Merchant Server
+    participant PP(SDK) as Paypal JS SDK
+    participant PP(ORDER) as Paypal Orders API
+    M(HTML)->>PP(SDK): Initialize PayPal Button
+    PP(SDK)->>M(HTML): Render PayPal Button
+    Buyer->>PP(SDK): Click PayPal Button (start checkout)
+    PP(SDK)->>M(HTML): createOrder() callback
+    M(HTML)->>M(S): Create Order
+    M(S)->>PP(ORDER): POST v2/orders
+    PP(ORDER)->>M(S): Order Created
+    M(S)->>M(HTML): Return order ID
+```
+
+<br/><br/>
+
+# 3. Change your `onApprove()` callback
+
+## Current state of client-side integration behavior
+
+```js
+onApprove(data, actions) {
+    return actions.order.capture()
+        .then(function (orderData) {
+          // Successful capture!
+        })
+        .catch(function (err) {
+          // Failed capture
+    });
+}
+```
+
+### We pass the OrderID for you to the helper
+On the client-side, the SDK provides an actions.orders.capture() helper for capturing the order, which takes care of passing the Order ID to the v2 Orders API capture endpoint and handling the authorization.
+
+### We automatically restart the flow for INSTRUMENT_DECLINED errors
```

Comment:
Is this section meant to be blank? Seems odd as a heading if theres no content to it